craft_parts/executor/environment.py

```diff
@@ -160,6 +160,8 @@ def _get_global_environment(info: ProjectInfo) -> dict[str, str]:
         "CRAFT_ARCH_TRIPLET_BUILD_FOR": info.arch_triplet_build_for,
         "CRAFT_PARALLEL_BUILD_COUNT": str(info.parallel_build_count),
         "CRAFT_PROJECT_DIR": str(info.project_dir),
+        # Build system configurations for the project.
+        "CARGO_HOME": str(info.dirs.work_dir / ".cargo"),
```

Comment:
for these internal directories this could be /cargo, this path is changing anyways, but something to consider.

Reply:
I prefer it un-hidden :-)